src/main/java/com/interswitchng/ewardrobe/controller/user/UserController.java

```diff
@@ -13,8 +19,13 @@
 public class UserController {
     private final UserService userService;
 
-    @PostMapping("/test")
-    public ResponseEntity <?> doSumthing(){
-        return null;
+    @PostMapping("/sign-up")
+    public ResponseEntity <?> signUp(@RequestBody SignupDto signupDto) throws UserAlreadyExistException, PasswordMisMatchException, InvalidEmailException {
+        try {
+            userService.signUp(signupDto);
+            return new ResponseEntity<>("User created successfully", HttpStatus.CREATED);
```

Comment:
Return a dto instead of a string, you can add essential fields to the dto like firstname, lastname etc.